app/scripts/fetch_xulrunner

```diff
@@ -284,6 +284,9 @@ function modify_omni {
 	# Hide "Run on sites with restrictions" row and helper message
 	echo '.addon-detail-row-quarantined-domains { display: none; }' >> $file
 	echo '.addon-detail-help-row:has([data-l10n-id="addon-detail-quarantined-domains-help"]) { display: none; }' >> $file
+	# Hide "Report" button from plugins context menu
+	echo '[data-l10n-id="report-addon-button"] { display: none }' >> $file
+	echo 'addon-card addon-options hr:first-of-type { display: none }' >> $file
```

Comment:
The `hr` is above the button, right? If so, should move that line above the other.

Reply:
Good point, thank you. I re-pushed that commit with the lines swapped